src/Microsoft.VisualStudio.Services.Agent/Constants.cs

```diff
@@ -38,7 +38,7 @@ public enum OSPlatform
 
         public static class Agent
         {
-            public static readonly string Version = "2.103.1";
+            public static readonly string Version = "2.104.0";
```

Comment:
OK.  We'll branch for RC1 off of 2.104.0.  We'll want to take a VSO change tomorrow in releases 103 pointing agent to 104.